### ISSUE TITLE
Unable to upload

### ISSUE THREAD

**Issue by jfeldstein** [heyupdate/sqwack#7]
I think I typed my password wrong. 

Now I'm unable to try again. It does not ask for password. 

How do I clear the session it's using?

**Comment by tompedals** [comment]
I don't think this is working anymore since Slack made changes to the photo upload form. It needs updating to work with this new form structure.

**Comment by tompedals** [comment]
However, you should be able to clear files in `~/.sqwack`
